core/include/userver/clients/http/form.hpp

```diff
@@ -6,11 +6,13 @@
 USERVER_NAMESPACE_BEGIN
 
 namespace curl {
+#if LIBCURL_VERSION_NUM <= 0x074700
 class form;
+#endif 
 }  // namespace curl
 
 namespace clients::http {
-
+#if LIBCURL_VERSION_NUM <= 0x074700
 class Form final {
```

Comment:
I think we have a few thousands usages of Form in our internal codebase. It can not be removed, but its implementation should be transparent ly replaced.